### PR TITLE
I'm a teapot

### PR DESCRIPTION
**Pull request by wheresrhys**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -72,6 +72,10 @@ module.exports = function(options) {
 		res.set({ 'Cache-Control': 'max-age=60' });
 		res.json(sensuChecks);
 	});
+	
+	app.get('/__teapot', function(req, res) {
```
Comment (ironsidevsquincy) [inline main.js]: `/__brew-coffee`?

Comment (wheresrhys) [inline main.js]: of course! I can never remember [obscure IT infrastructure protocol] jokes

**Comment by keirog** [comment]
Yessss

**Comment by charypar** [comment]
:coffee:

File: main.js [hunk 2]
```diff
@@ -72,6 +72,10 @@ module.exports = function(options) {
 		res.set({ 'Cache-Control': 'max-age=60' });
 		res.json(sensuChecks);
 	});
+	
+	app.get('/__brew-coffee', function(req, res) {
```
Comment (matthew-andrews) [inline main.js]: this should be tea e.g. `/__yorkshire` or at the very least `/__brew`.

File: main.js [hunk 3]
```diff
@@ -72,6 +72,10 @@ module.exports = function(options) {
 		res.set({ 'Cache-Control': 'max-age=60' });
 		res.json(sensuChecks);
 	});
+
```
Comment (matthew-andrews) [inline main.js]: unexpected spaces.

File: main.js (hunk 2 above)
Comment (wheresrhys) [inline main.js]: nah the status is in the class of 'bad request' errors

**Pull request merged**